### DISCUSSION TITLE
添加下载光源的功能，和优化联机模块。

### DISCUSSION THREAD

**Discussion by tou002** [Meloong-Git/PCL#205] [提问] [answered]
每次光影都要到网上找太麻烦了，而且下载速度某某网盘不氪金限速100kb/s

**Comment by Big-Cake-jpg** [comment, marked as answer]
国内尚未有统一的、合理的下载途径，且 CurseForge 在国内的连通性可能较差。

以及，新功能建议请提交 Issue。
